notes/2.md

```diff
@@ -68,6 +68,79 @@ Set up scripts
   }
 }
 ```
+Create empty `gatsby-config.js` in www directory
+![image](https://user-images.githubusercontent.com/3941856/74722809-63ffdb80-51ee-11ea-97a3-910159021ac8.png)
+
+```javascript
+module.exports = {}
+```
+Create www/src/pages/index.js
+![image](https://user-images.githubusercontent.com/3941856/74723055-cc4ebd00-51ee-11ea-9655-d3cc51981452.png)
+
+```javascript
+export default props => (
+  <div>
+    <h1>Our Site</h1>
+  </div>
+);
+```
+
+Go back to the root of the project and run
+
+```
+yarn workspace www develop
+```
+
+Get an error, import React in index.js to fix it
+
+```javascript
+import React from "react";
+export default props => (
+  <div>
+    <h1>Our Site</h1>
+  </div>
+);
+```
+
+Add packages directory, package.json, and yarn.lock to git 
+
+```
+ga package* yarn.lock
+```
+
+This adds the .cache folder and we don't want to check that in
+
+```
+git reset packages/www/.cache
+```
+
+Create `.gitignore` in the project root, add .cache
+![image](https://user-images.githubusercontent.com/3941856/74724047-7da22280-51f0-11ea-8b42-937cad070677.png)
+
+Add `.gitignore`
+
+```
+ga .gitignore
+```
+
+Add the public folder to `.gitignore`
+
+Contents now look like this:
+```
+.cache
+public
+```
+
+Now add only the files we want to track, commit, and push
+
+```
+ga -u
```

Comment:
https://git-scm.com/docs/git-add

-u
--update

Update the index just where it already has an entry matching `<pathspec>`. This removes as well as modifies index entries to match the working tree, but adds no new files.

If no `<pathspec>` is given when -u option is used, all tracked files in the entire working tree are updated (old versions of Git used to limit the update to the current directory and its subdirectories).


`git add --update`